packages/fx-core/src/plugins/resource/bot/plugin.ts

```diff
@@ -188,60 +191,70 @@ export class TeamsBotImpl {
     return ResultFactory.Success();
   }
 
-  public async updateArmTemplates(context: PluginContext): Promise<FxResult> {
+  public async updateArmTemplates(ctx: PluginContext): Promise<FxResult> {
     Logger.info(Messages.UpdatingArmTemplatesBot);
-
+    const azureSolutionSettings = ctx.projectSettings?.solutionSettings as AzureSolutionSettings;
+    const plugins = getActivatedV2ResourcePlugins(azureSolutionSettings).map(
+      (p) => new NamedArmResourcePluginAdaptor(p)
+    ); // This function ensures return result won't be empty
+    const pluginCtx = { plugins: plugins.map((obj) => obj.name) };
     const bicepTemplateDir = path.join(getTemplatesFolder(), PathInfo.BicepTemplateRelativeDir);
-
+    let configModule = await fs.readFile(
+      path.join(bicepTemplateDir, PathInfo.ConfigurationModuleTemplateFileName),
```

Comment:
these two calls can extract into one method? Since I saw lots of places have continuous two calls of these two methods.

Reply:
thanks!
updated.